docs/retired.rst

```diff
@@ -0,0 +1,68 @@
+#########################################
+Retired systems
+#########################################
+
+The following systems have been retired. For the full list of available systems, run `service.backends()`.  For more information, see  `Run on quantum backends <how_to/backends.html>`__.
+
+To retrieve results from retired backends, use `qiskit-ibm-provider <https://qiskit.org/documentation/partners/qiskit_ibm_provider/stubs/qiskit_ibm_provider.IBMBackend.html#qiskit_ibm_provider.IBMBackend>` or `QiskitRuntimeService.jobs <https://qiskit.org/documentation/partners/qiskit_ibm_runtime/stubs/qiskit_ibm_runtime.QiskitRuntimeService.jobs.html>`__.
+
+   
++-----------------------+----------------------+
+| System name           | Qubit count          |
++=======================+======================+
+| **Montreal**          | **27**               |
++-----------------------+----------------------+
+| **Toronto**           | **27**               |
++-----------------------+----------------------+
+| Armonk                | 1                    |
```

Comment:
Can we use the real system names, i.e. `ibmq_armonk`, instead of just the city name?

Reply:
We've always just used city names; I'll investigate and update if everyone agrees that's OK.

Reply:
updated